cmd/runner/taskrunner/taskrunner.go

```diff
@@ -83,18 +83,18 @@ func New(cfg *Config) (*TaskRunner, error) {
 }
 
 func (tr *TaskRunner) ParsePortfolio(ctx context.Context, req *task.ParsePortfolioRequest) (task.ID, task.RunnerID, error) {
-	var buf bytes.Buffer
-	if err := json.NewEncoder(&buf).Encode(req.AssetIDs); err != nil {
-		return "", "", fmt.Errorf("failed to encode asset IDs: %w", err)
+	var taskBuffer bytes.Buffer
```

Comment:
`*twitches in Go*`: https://github.com/golang/go/wiki/CodeReviewComments#variable-names

For a short function like this, `buf` is unambiguous and to the point

Reply:
I care less about this than you do, so changed.

I haven't yet bought into the short var name philosophy of go - it seems like just a different preference.